server/datastore/mysql/migrations/tables/20230721161508_QueriesDataMigrator_test.go

```diff
@@ -0,0 +1,294 @@
+		automationsEnabled = append(automationsEnabled, query.AutomationsEnabled)
+		loggingTypes = append(loggingTypes, query.LoggingType)
+
+		require.Equal(t, query.Description, "Admin desc")
+		require.Equal(t, query.Query, "SELECT * FROM osquery_info;")
+		require.Equal(t, *query.AuthorID, uint(1))
+		require.Equal(t, query.Saved, true)
+		require.Equal(t, query.ObserverCanRun, true)
+	}
+	require.ElementsMatch(t, names, []string{"Admin Global Query - 1", "Admin Global Query - 2"})
+	require.ElementsMatch(t, scheduleIntervals, []uint{3600, 86400})
+	require.ElementsMatch(t, automationsEnabled, []bool{true, true})
+	require.ElementsMatch(t, loggingTypes, []string{"snapshot", "snapshot"})
+	require.Equal(t, nRows, 2)
+
+	rows, err = db.Query(stmt, "per_query_perf%")
+	require.NoError(t, err)
+	defer rows.Close()
+
+	nRows = 0
+	names = []string{}
+	scheduleIntervals = []uint{}
+	automationsEnabled = []bool{}
+	loggingTypes = []string{}
+
+	for rows.Next() {
+		nRows += 1
+		query := fleet.Query{}
+		err := rows.Scan(
+			&query.Name,
+			&query.Description,
+			&query.Query,
+			&query.AuthorID,
+			&query.Saved,
+			&query.ObserverCanRun,
+			&query.Platform,
+			&query.MinOsqueryVersion,
+			&query.ScheduleInterval,
+			&query.LoggingType,
+			&query.AutomationsEnabled,
+		)
+		require.NoError(t, err)
+
+		names = append(names, query.Name)
+		scheduleIntervals = append(scheduleIntervals, query.ScheduleInterval)
+		automationsEnabled = append(automationsEnabled, query.AutomationsEnabled)
+		loggingTypes = append(loggingTypes, query.LoggingType)
+
+		require.Equal(t, query.Description, "Records the CPU time and memory usage for each individual query. Helpful for identifying queries that may impact performance.")
+		require.Equal(t, query.Query, "SELECT name, interval, executions, output_size, wall_time, (user_time/executions) AS avg_user_time, (system_time/executions) AS avg_system_time, average_memory FROM osquery_schedule;")
+		require.Equal(t, *query.AuthorID, uint(1))
+		require.Equal(t, query.Saved, true)
+		require.Equal(t, query.ObserverCanRun, false)
+	}
+	require.ElementsMatch(t, names, []string{"per_query_perf", "per_query_perf - 7", "per_query_perf - 8"})
+	require.ElementsMatch(t, scheduleIntervals, []uint{0, 86400, 86400})
+	require.ElementsMatch(t, automationsEnabled, []bool{false, true, true})
+	require.ElementsMatch(t, loggingTypes, []string{"", "snapshot", "snapshot"})
+	require.Equal(t, nRows, 3)
+
+	// We have two team packs (Team-1, Team-2)
+	// For Team-1, we have three schedules, two of them reference 'User 1 Query', the last one
+	// 'per_query_perf', so I expect to see five different queries on team#1:
+	//   - 'User 1 Query - 9' for schedule#9
+	//   - 'User 1 Query - 10' for schedule#10
+	//   - 'per_query_perf - 11' for schedule#11
+	// For Team-2, we only have one schedule on 'backup_tool_perf', so I expect to see on team#2:
+	// 	 - 'backup_tool_perf - 12'
```

Comment:
Looks like the tests only asserts the one `per_query_perf` query exists, not the other 3?

Reply:
Yeah, the idea that I had was to only test for the one that belong to a team pack here.